github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/trigger/check/BranchToCauseConverter.java

```diff
@@ -24,65 +23,76 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(BranchToCauseConverter.class);
 
     private final GitHubBranchRepository localRepo;
-    private final LoggingTaskListenerWrapper listener;
+    private final LoggingTaskListenerWrapper pollingLog;
```

Comment:
in PR Trigger it's listener AFAIR, in jenkins it listener everywhere (fact that listener attached to file is other case).